templates/sync-catalog-deployment.yaml

```diff
@@ -117,6 +123,18 @@ spec:
                 {{- if .Values.syncCatalog.addK8SNamespaceSuffix}}
                 -add-k8s-namespace-suffix \
                 {{- end}}
+                {{- if .Values.global.consulNamespacesEnabled }}
+                -enable-namespaces=true \
+                {{- if .Values.syncCatalog.consulNamespaces.consulDestinationNamespace }}
+                -consul-namespace={{ .Values.syncCatalog.consulNamespaces.consulDestinationNamespace }} \
```

Comment:
Looking at the old options, we had:
* `-k8s-service-prefix` - `syncCatalog.k8sPrefix`
* `-k8s-source-namespace` - `syncCatalog.k8sSourceNamespace`
* `-k8s-write-namespace` - `${NAMESPACE}` (comes from the release namespace)
* `-consul-service-prefix`- `syncCatalog.consulPrefix`

We're now adding:
* `-consul-namespace` - `consulNamespaces.consulDestinationNamespace`
* `-enable-namespace-mirroring` - `consulNamespaces.mirrorK8S`
* `-mirroring-prefix` - `consulNamespaces.mirroringPrefix`

Based on the old naming conventions, these should actually be:
* `-consul-write-namespace` - `consulNamespaces.consulWriteNamespace`
* `-enable-k8s-namespace-mirroring` - `consulNamespaces.mirrorK8S` (Specifying this is k8s to consul in the flag is important if we ever want to support consul to k8s in the future)
* `-k8s-namespace-mirroring-prefix` - `consulNamespaces.mirroringPrefix`

So to sum up the changes:
* `-consul-namespace` => `-consul-write-namespace`
* `consulNamespaces.consulDestinationNamespace` => `consulNamespaces.consulWriteNamespace`
* `-enable-namespace-mirroring` => `-enable-k8s-namespace-mirroring`
* `-mirroring-prefix` => `-k8s-namespace-mirroring-prefix`

What do you think?